### ISSUE TITLE
Add Regular Expression Handler to @activities.slash

### ISSUE THREAD

**Issue by ystxn** [finos/symphony-bdk-python#255]
The Java BDK has supported regular expressions in its @Slash handler since the start but the Python BDK did not. This creates a discrepancy when describing how to build command handlers that handle more complex use-cases such as catch-all scenarios between both languages.

```java
// Valid in Java BDK
bdk.activities().register(slash("/Help .*", false, context -> {
    // Logic
}));
```

```python
# Invalid in Python BDK
@activities.slash("/Help .*")
async def hello(context: CommandContext):
    # Logic
```

**Comment by symphony-youri** [comment]
We could address this point along with #238

**Comment by dky** [comment]
+1 for this, would love to try this if this feature gets implemented.

**Comment by ystxn** [comment]
Closed with #238 bringing parity of command arguments to Python as in the Java BDK
